Source/WebKit/NetworkProcess/cocoa/AppStoreDaemonSPI.h

```diff
@@ -60,4 +60,17 @@ typedef NS_ENUM(NSInteger, ASDInstallWebAttributionContext) {
 NS_ASSUME_NONNULL_END
 
 #endif // USE(APPLE_INTERNAL_SDK)
+
+// FIXME: Move these to the !USE(APPLE_INTERNAL_SDK) section once rdar://137446922 is complete.
+#if HAVE(AD_ATTRIBUTION_KIT_PRIVATE_BROWSING)
+NS_ASSUME_NONNULL_BEGIN
+@interface ASDInstallWebAttributionService (Staging_for_137446922)
```

Comment:
This "staging for" thing is news to me. What does it control or convey?

Reply:
Once that radar is completed and included in an SDK we can move this to the section of the SPI header that is only for non-internal SDKs.  Until then we need a selector declaration when building with internal and non-internal SDKs.